### ISSUE TITLE
perf-config documentation and usage

### ISSUE THREAD

**Issue by itsul** [mickem/nscp#363]
Hi Michael,

we tried to change the default perf-data output without success.

[root@vm nrpe-master]# /usr/local/nagios/libexec/check_nrpe -H 10.10.10.10
I (0.5.0.62 2016-09-14) seem to be doing fine...

[root@vm nrpe-master]# /usr/local/nagios/libexec/check_nrpe -H 10.10.10.10 -c check_cpu -a "warning=load > 80%" "critical=load > 90%" time=1m time=5m time=15m "ok-syntax=%(status): CPU load is ok." "detail-syntax=CPU load is %(time):%(load)%" "perf-syntax=%(time)" "perf-config=%(core)0,100"
OK: CPU load is ok.|'1m'=15%;80;90 '5m'=15%;80;90 '15m'=14%;80;90

Could you add the documentation for "perf-config" (cpu,disk, memory) please? Furthermore qgarnier from centreon asked for min/max values in the perf-data #https://github.com/mickem/nscp/issues/334 we would appreciate that function too. It is usefull for generating reports.

Regards

**Comment by mickem** [comment]
Have you checked the documentation here: https://docs.nsclient.org/checks/#performance-data-configuration

As for min/max it is difficult in som instances and it is general avalible when applicable.
For instance:

```
check_drivesize
L        cli CRITICAL: CRITICAL C:\: 437.05GB/452.338GB used
L        cli  Performance data: 'C:\ used'=437.05017GB;361.8706;407.10443;0;452.33826...
```

For % it makes no sense as the maximum is always 100 and the report should know as the unit is set to %.

I guess I could do a quick check_add_minmac wrapper which adds them in?

**Comment by itsul** [comment]
Hi,

if you would add the wrapper that will be great. Otherwise we were thinking to add a wrapper as a single script, but build in is much better.

The docu was already checked and you were right, that the min/max is added to storage checks but not for cpu and other checks.

**Comment by mickem** [comment]
Is there anything I can improve in terms of documentation?


As for the perf thing next version will have new mode in xform_perf like so:
```
xform_perf mode=minmax command=check_cpu
L        cli OK: OK: CPU load is ok.
L        cli  Performance data: 'total 5m'=0%;80;90;0;100 'total 1m'=0%;80;90;0;100 'total 5s'=5%;80;90;0;100
```

**Comment by itsul** [comment]
Hi Michael,

for the moment the documentation is good. When the xform_perf is available the problem is solved.

Regards

**Comment by mickem** [comment]
The command is workaround I described is available in the latest 0.5.1 version.
